quinn-udp/src/unix.rs

```diff
@@ -833,6 +844,118 @@ mod gso {
     pub(crate) fn set_segment_size(encoder: &mut cmsg::Encoder<libc::msghdr>, segment_size: u16) {
         encoder.push(libc::SOL_UDP, UDP_SEGMENT, segment_size);
     }
+
+    fn supported_by_current_kernel() -> bool {
+        let kernel_version_string = match kernel_version_string() {
+            Ok(kernel_version_string) => kernel_version_string,
+            Err(_errno) => {
```

Comment:
Why do these need `_` prefixes?

Reply:
When the log macros are configured out (no features active), they'd otherwise show up as unused variables.